addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -204,11 +204,11 @@
 
 <template id="account_invoice_it_FatturaPA_sede">
             <Sede>
-                <Indirizzo><t t-if="partner.street" t-esc="format_alphanumeric(partner.street)"/> <t t-if="partner.street2" t-esc="format_alphanumeric(partner.street2)"/></Indirizzo>
-                <CAP><t t-if="partner.country_id.code != 'IT'" t-esc="'00000'"/><t t-else="" t-esc="partner.zip"/></CAP>
-                <Comune t-esc="format_alphanumeric(partner.city)"/>
-                <Provincia t-if="partner.country_id.code == 'IT'" t-esc="partner.state_id.code"/>
-                <Nazione t-esc="partner.country_id.code"/>
+                <Indirizzo><t t-if="partner.street or partner.street2" t-esc="format_alphanumeric((partner.street or '') + ' ' + (partner.street2 or ''))[:60]"/></Indirizzo>
+                <CAP><t t-if="partner.country_id.code != 'IT'" t-esc="'00000'"/><t t-elif="partner.zip" t-esc="partner.zip[:5]"/></CAP>
```

Comment:
This is in the general validations already

Reply:
ROTFL 🤣  That's true!
And why just this and the VAT fields are checked in the code?
![image](https://user-images.githubusercontent.com/1665365/208961709-5e56e39b-b71c-476c-9e5d-d3e39e587c26.png)
I'm removing it then.